issue-creator/cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: CronJob
```

Comment:
@kargakis Getting the collaborators on a repository requires push access to the repository. 😞  I could stop validating the assignees but issues will fail to be created sometimes because there are non-collaborators in the `test-owners.csv` file.